pkg/cli/image/manifest/manifest.go

```diff
@@ -42,12 +42,14 @@ type SecurityOptions struct {
 	RegistryConfig   string
 	Insecure         bool
 	SkipVerification bool
+	ForcePrefix      bool
 
 	CachedContext *registryclient.Context
 }
 
 func (o *SecurityOptions) Bind(flags *pflag.FlagSet) {
 	flags.StringVarP(&o.RegistryConfig, "registry-config", "a", o.RegistryConfig, "Path to your registry credentials (defaults to ~/.docker/config.json)")
+	flags.BoolVar(&o.ForcePrefix, "force-prefix", o.ForcePrefix, "Force lookup of named prefix (registry/repository/name) for an image source")
```

Comment:
It feels strange to me to have this flag down on `oc image info ...` and such but have handling only up under `oc adm release ...`.

Reply:
yes, wasn't sure where to place the flag.  I was going to take suggestions.  I'm reworking this to use the ImageContentSourcePolicy information, though, and possibly remove the need for a user to pass a flag at all - working on that now